i18n/de.toml

```diff
@@ -169,7 +169,7 @@ label = "Max. Ladestrom"
 label = "Min. Ladestrom"
 
 [main.loadpointSettings.minSoc]
-description = "Reichweite für Notfälle. Fahrzeug wird im PV-Modus „schnell” auf {0}% geladen. Danach weiter mit PV-Überschuss."
+description = "Reichweite für Notfälle. Fahrzeug wird im PV-Modus „schnell“ auf {0} % geladen. Danach weiter mit PV-Überschuss."
```

Comment:
Good Catch mit dem Apostroph- falls wir diese wirklich nutzen wollen sollten wir das aber konsistent über alle Sprachen tun.

Hier wiederrum: PV-Überschuss mit Bindestrich.

Reply:
@andig

- Im Deutschen werden typografische Anführungszeichen nach dem „99-66-Schema“ gesetzt: https://www.duden.de/sprachwissen/rechtschreibregeln/anfuehrungszeichen, https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen#Typografische_Anf%C3%BChrungszeichen
- Der Bindestrich bei „PV-Überschuss“ passt hier schon; siehe: https://www.duden.de/sprachwissen/rechtschreibregeln/bindestrich#D28
- Wegen des Leerzeichens zwischen Zahl und `%` siehe: https://github.com/evcc-io/evcc/pull/8480#discussion_r1233007182

Reply:
% Diskussion siehe oben

Reply:
Anführungszeichen in Weblate angepasst. Abstand findet in eigenem Issue statt.